crates/build-rs/src/input.rs

```diff
@@ -102,6 +102,13 @@ mod cfg {
     // those disabled with #[cfg(any())] don't seem meaningfully useful
     // but we list all cfg that are default known to check-cfg
 
+    /// Each activated feature of the package being built
+    #[doc = requires_msrv!("1.85")]
```

Comment:
This will need to be updated based on when this gets merged

Reply:
It'd probably be beneficial to set up a `CURRENT_RUST_VERSION` placeholder like is used for stabilization in rustc

Reply:
Yes, we only recently started tracking these situations more regularly in 1.81 and haven't looked at updating our processes yet.  So far, this has been working out ok.